components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -259,7 +259,7 @@
                             <a class="step_order_header" href="#">Step <%=step.getStepOrder()%>
                             </a>
                             <a href="#" class="delete_step icon-link"
-                               style="background-image: url(images/delete.gif);float:right;width: 9px;"></a>
+                               style="background-image: url(images/delete.gif);float:right;width: 9px;" data-step-no="<%=step.getStepOrder()%>"></a>
```

Comment:
Is this change related to this PR? There are some other files as well, Please check.

Reply:
It seems like the branch `wso2:feature-sp-from-vcs` is not rebased with the latest master as the diff between my fork branch and `wso2:master` doesn't contain that file.

(https://github.com/wso2/carbon-identity-framework/compare/master...kaveenr:remotefetch-feature)

Reply:
When you are rebasing with the wso2:master, you can skip other comments.